ntpd is failing to start

```
2019/09/07 23:15:38.725828 main.go:59: Starting ntpd
2019/09/07 23:15:38.726075 ntp.go:47: error querying pool.ntp.org for time, failed to get a response back from ntp server after 0 retries
2019/09/07 23:15:38.726159 main.go:73: failed to get a response back from ntp server after 0 retries
```

--- a/internal/app/ntpd/main.go
+++ b/internal/app/ntpd/main.go
@@ -54,7 +54,12 @@ func main() {
 		server = data.Services.NTPd.Server
 	}
 
-	n := &ntp.NTP{Server: server}
+	n, err := ntp.NewNTPClient(
+		ntp.WithServer(server),
+	)
+	if err != nil {
+		log.Fatalf("failed to create ntp client: %v", err)
+	}
 
 	log.Println("Starting ntpd")
 	errch := make(chan error)

